src/client/BugsnagTelemetrySender.ts

```diff
@@ -0,0 +1,23 @@
+// The Ed-Fi Alliance licenses this file to you under the Ed-Fi License Agreement.
+// See the LICENSE file in the project root for more information.
+
+// eslint-disable-next-line import/no-unresolved
+import { TelemetrySender, env, extensions } from 'vscode';
+import bugsnag from '@bugsnag/js';
+
+bugsnag.start({
+  apiKey: '935aa2343fcce2db62e77c03c2080251',
```

Comment:
The presence of an API key is a surprise to me. I'm not sure that we really want to distribute something with an API key; I need to think about that before merging this pull request.

If we go ahead with this merge, we'll want to store the API key as a GitHub secret and inject it at compile time, so that at least it is "hidden" in the VSIX-packaged code.

Reply:
Here's BugSnag's response to a question about distributing the API key: https://github.com/bugsnag/bugsnag-js/issues/595

Seeing that it was already in Atom code and we haven't had a problem, let's go ahead... but again, before merging as is, let's think about how we could inject this at build time. Maybe in the workflow a simple [sed command](https://linuxize.com/post/how-to-use-sed-to-find-and-replace-string-in-files/) to find and replace a token with the correct key.

With the old Atom code this wasn't as big a deal since that was closed source, but given that this is an open source repo, I'm more sensitive to the disclosure issue, even if the scope of what can be done with the key is very small.

Reply:
I working on something for this